tests/BuildScriptGenerator.Tests/DotnetCore/DotNetCorePlatformTest.cs

```diff
@@ -19,82 +19,41 @@ public class DotNetCorePlatformTest
         public void Detect_ThrowsUnsupportedException_ForUnknownNetCoreAppVersion()
         {
             // Arrange
-            var projectFile = "test.csproj";
-            var projectFileContent = SampleProjectFileContents.ProjectFileWithTargetFrameworkPlaceHolder.Replace(
-                "#TargetFramework#",
-                "netcoreapp0.0");
-            var sourceRepo = new Mock<ISourceRepo>();
-            sourceRepo
-                .Setup(repo => repo.EnumerateFiles(It.IsAny<string>(), It.IsAny<bool>()))
-                .Returns(new[] { projectFile });
-            sourceRepo
-                .Setup(repo => repo.ReadFile(It.IsAny<string>()))
-                .Returns(projectFileContent);
-            var context = CreateContext(sourceRepo.Object);
-            var platform = CreatePlatform(projectFile: projectFile, defaultVersion: "2.2.7");
-
-            var exception = Assert.Throws<UnsupportedVersionException>(
-                () => platform.Detect(context));
-            Assert.Equal(
-                $"Platform 'dotnet' version '0.0' is unsupported. " +
-                "Supported versions: 2.2.7",
-                exception.Message);
-        }
+            var detector = CreateDetector(detectedVersion: "0.0");
```

Comment:
Looks like some tests are removed here, please make sure they're covered either in detector tests or here in the generator tests

Reply:
Sure, will do. But to give some context what the change I made here: Earlier this test used to depend on the detector (present in BuildScriptGenerator) to parse the `netcoreapp0.0` and then throw the exception, but now since Detector library took the responsibility of detecting the version, this test is only supplying the `detectedVersion` directly and is expecting the exception to be thrown.